src/models/reactions.rs

```diff
@@ -0,0 +1,31 @@
+use super::*;
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub enum ReactionContent {
+    #[serde(rename = "heart")]
+    Heart,
+    #[serde(rename = "+1")]
+    PlusOne,
+    #[serde(rename = "laugh")]
+    Laugh,
+    #[serde(rename = "confused")]
+    Confused,
+    #[serde(rename = "hooray")]
+    Hooray,
+    #[serde(rename = "-1")]
+    MinusOne,
+    #[serde(rename = "rocket")]
+    Rocket,
+    #[serde(rename = "eyes")]
+    Eyes,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[non_exhaustive]
+pub struct Reaction {
+    pub id: ReactionId,
+    pub node_id: String,
+    pub user: User,
```

Comment:
This might be optional, no? According to a schema https://docs.github.com/en/rest/reactions#create-reaction-for-an-issue:

```
"user": {
      "anyOf": [
        {
          "type": "null"
        },
        {
          "title": "Simple User",
...